src/Servers/testassets/TestSites/ServerComparison.TestSites.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
+    <StandardTestTfms>netcoreapp2.1;netcoreapp2.0</StandardTestTfms>
+    <StandardTestTfms Condition=" '$(DeveloperBuild)' != 'true' AND '$(OS)' == 'Windows_NT' ">$(StandardTestTfms);net461</StandardTestTfms>
+
+    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
```

Comment:
This re-defines what is already set in line 4. You can remove lines 5-8 in favor of line 4.